services/libs/data-access-layer/src/orgs/index.ts

```diff
@@ -0,0 +1,49 @@
+import { IOrganizationPartialAggregatesRawResult } from './types'
+import { QueryExecutor } from '../queryExecutor'
+
+export async function findOrgsForMergeSuggestions(
+  qx: QueryExecutor,
+  tenantId: string,
+  batchSize: number,
+  afterOrganizationId?: string,
+  lastGeneratedAt?: string,
+): Promise<IOrganizationPartialAggregatesRawResult[]> {
+  let filter = ''
+  if (afterOrganizationId) {
+    filter += `AND o.id > $(afterOrganizationId)`
+  }
+
+  if (lastGeneratedAt) {
+    filter += 'AND o."createdAt" > $(lastGeneratedAt)'
+  }
+
+  return qx.select(
+    `
+      SELECT
+        o.id,
+        json_agg(oi) as identities,
+        ARRAY_AGG(DISTINCT onm."noMergeId") AS "noMergeIds",
+        o."displayName",
+        o.location,
+        o.industry,
+        o.website,
```

Comment:
There is no `website` column anymore. It is moved as an identity with type `primary-domain` into `organizationIdentities` table.